template/bison/yacc.c

```diff
@@ -101,8 +101,12 @@
 # endif
 
 <%# b4_header_include_if -%>
+<%- if output.include_header -%>
+#include "<%= output.include_header %>"
+<%- else -%>
 /* Use api.header.include to #include this header
    instead of duplicating it here.  */
+<%- end -%>
```

Comment:
Oh, I overlooked the location of `<%- end -%>` is wrong...
I will fix it within https://github.com/ruby/lrama/pull/135.

Reply:
Oh, my apologies. Thank you for fixing it 🙇🏻